govcd/org_test.go

```diff
@@ -80,8 +80,14 @@ func (vcd *TestVCD) Test_DeleteOrg(check *C) {
 	err = org.Delete(true, true)
 	check.Assert(err, IsNil)
 	// Check if org still exists
-	time.Sleep(3 * time.Second)
-	org, err = GetAdminOrgByName(vcd.client, TestDeleteOrg)
+	for i := 0; i < 30; i++ {
+		org, err = GetAdminOrgByName(vcd.client, TestDeleteOrg)
+		if org == (AdminOrg{}) {
+			break
+		} else {
+			time.Sleep(1 * time.Second)
+		}
+	}
```

Comment:
There are three occurrences of the same loop in this commit. Please think of a good way to get rid of these duplications. I mean something between the lines of:
- What about introducing a utility function in `system.go` which would wait for the org for the given amount of seconds. Something like `WaitAdminOrgByName(vcd.client, TestDeleteOrg, 30)`.
- Or what about introducing a task here, if it takes so long to be created?
- Etc.

Reply:
This is test, so no task here. We using old api part where no task returned and changes to Organization API not this PR part. I introduced sleep just I spotted that it randomly fails when delete wasn't fast at some time.

Reply:
OK, I now realise that you are actually waiting for Delete to completely _remove_ the Org.